stdlib/Distributed/src/cluster.jl

```diff
@@ -746,8 +769,18 @@ end
 """
     nworkers()
 
-Get the number of available worker processes. This is one less than `nprocs()`. Equal to
+Get the number of available worker processes. This is one less than [`nprocs()`](@ref). Equal to
 `nprocs()` if `nprocs() == 1`.
+
+# Examples
+Here, `julia` was launched with `-p 5`:
```

Comment:
Since these are `julia-repl` blocks, maybe we can instead do something like
````
$ julia -p 5

julia> nprocs()
6

nworkers()
5
```
? Ans similarly on the other places.